homeassistant/components/hydrawise/__init__.py

```diff
@@ -5,15 +5,23 @@
 from requests.exceptions import ConnectTimeout, HTTPError
 import voluptuous as vol
 
-from homeassistant.components import persistent_notification
-from homeassistant.const import CONF_ACCESS_TOKEN, CONF_SCAN_INTERVAL
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.const import (
+    CONF_ACCESS_TOKEN,
+    CONF_API_KEY,
+    CONF_SCAN_INTERVAL,
+    Platform,
+)
 from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.typing import ConfigType
 
-from .const import DOMAIN, LOGGER, NOTIFICATION_ID, NOTIFICATION_TITLE, SCAN_INTERVAL
+from .const import DOMAIN, LOGGER, SCAN_INTERVAL
 from .coordinator import HydrawiseDataUpdateCoordinator
 
+# Deprecated since Home Assistant 2023.8.0
+# Can be removed completely in 2024.2.0
```

Comment:
```suggestion
# Deprecated since Home Assistant 2023.9.0
# Can be removed completely in 2024.3.0
```
These comments could be removed as we have the issues but you need to change the version there

Reply:
Removed the comments.